code/game/machinery/suit_storage_unit.dm

```diff
@@ -51,10 +52,6 @@
 	/// How fast it charges cells in a suit
 	var/charge_rate = 250
 
-/obj/machinery/suit_storage_unit/Initialize(mapload)
-	. = ..()
-	interaction_flags_machine |= INTERACT_MACHINE_OFFLINE
```

Comment:
this was added at some point. suit storage units have behavior to be opened up with a crowbar when unpowered so it makes no sense